packages/scene-composer/src/components/three-fiber/EditorTransformControls.spec.tsx

```diff
@@ -40,11 +40,9 @@ jest.doMock('../../../src/three/TransformControls', () => {
 });
 
 const mockFindComponentByType = jest.fn();
-const mockisEnvironmentNode = jest.fn();
 jest.doMock('../../../src/utils/nodeUtils', () => {
```

Comment:
nit: Not part of your CR, but when you see usages of `jest.doMock` we should refactor them to use `jest.mock` properly.

doMock doesn't hoist the mocking logic up, so we still end up with importing all the dependencies we are mocking, and it leads to inconsistencies and obscure test issues.